src/Codeception/Subscriber/GracefulTermination.php

```diff
@@ -1,22 +1,35 @@
 <?php
-declare (ticks = 1);
+
+declare(strict_types=1);
+declare(ticks=1);
+
 namespace Codeception\Subscriber;
 
 use Codeception\Event\SuiteEvent;
 use Codeception\Events;
+use RuntimeException;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use function function_exists;
+use function pcntl_async_signals;
+use function pcntl_signal;
 
 class GracefulTermination implements EventSubscriberInterface
 {
+    /**
+     * @var string
+     */
     const SIGNAL_FUNC = 'pcntl_signal';
+    /**
+     * @var string
+     */
     const ASYNC_SIGNAL_HANDLING_FUNC = 'pcntl_async_signals';
 
     /**
      * @var SuiteEvent
      */
     protected $suiteEvent;
 
-    public function handleSuite(SuiteEvent $event)
+    public function handleSuite(SuiteEvent $event): void
     {
         if (PHP_MAJOR_VERSION === 7 && PHP_MINOR_VERSION === 0) {
```

Comment:
Wow, it is still here, another piece of code to remove.

Reply:
It was removed in the other PR, you have not merged it yet